lib/gollum/views/page.rb

```diff
@@ -26,14 +26,14 @@ def page_header
       end
       
       def breadcrumb
-        path = Pathname.new(@page.url_path_title)
+        path = Pathname.new(@page.url_path).parent
```

Comment:
This is the fix proper (using `url_path` rather than `url_path_title`), the rest is some mild refactoring, taking advantage of `Pathname#parent` to make sure the page's title doesn't end up in the breadcrumb.